drivers/hmis/app/models/hmis/auto_exit_config.rb

```diff
@@ -0,0 +1,48 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+class Hmis::AutoExitConfig < Hmis::HmisBase
+  self.table_name = 'hmis_auto_exit_configs'
+
+  def matching_projects
+    scope = Hmis::Hud::Project.all
+
+    scope = scope.where(project_type: project_type) if project_type.present?
+    scope = scope.where(organization_id: Hmis::Hud::Organization.find_by(id: organization_id).organization_id) if organization_id.present?
+    scope = scope.where(project_id: Hmis::Hud::Project.find_by(id: project_id).project_id) if project_id.present?
+
+    scope
+  end
+
+  def self.all_projects_config
```

Comment:
For clarity, maybe call this `default_config`? It doesn't apply to all projects, since some projects may have more specific configs